src/FluentResults/Reasons/Error.cs

```diff
@@ -67,6 +67,18 @@ public Error CausedBy(Error error)
             return this;
         }
 
+        /// <summary>
+        /// Set the root cause of the error
+        /// </summary>
+        public Error CausedBy(IError error)
```

Comment:
Please don't add new methods. Only change the existing method with Error error to IError error.

Reply:
Done.